cmd/rollkit/commands/run_node_test.go

```diff
@@ -33,6 +33,7 @@ func TestParseFlags(t *testing.T) {
 		"--rollkit.da_mempool_ttl", "10",
 		"--rollkit.da_namespace", "namespace",
 		"--rollkit.da_start_height", "100",
+		"--rollkit.da_keyring_keyname", "my_celes_key",
```

Comment:
```suggestion
		"--rollkit.da_keyring_keyname", null,
```

I'm starting to think that it would be good to leave the default value null so that it only is used when someone intentionally sets the keyname. otherwise, it may accidentally redirect from key_node_is_running_with to my_celes_key default, instead of just sending requests to key_node_is_running_with. this flag is only going to be useful for those using multiaccounts on one light node / da service